packages/node/src/behaviors/level-control/LevelControlServer.ts

```diff
@@ -246,10 +246,14 @@ export class LevelControlServerLogic extends LevelControlLogicBase {
         withOnOff: boolean,
         options: TypeFromPartialBitSchema<typeof LevelControl.Options> = {},
     ) {
+        if (rate === 0) {
```

Comment:
This change and the OnOff one is in fact 1.4 changes. Dahlias we move out here and do separate pr when we do 1.4?

Reply:
discussed in discord, leaving as is